tools/workflow/clean_aws_resources.go

```diff
@@ -33,18 +33,25 @@ const (
 )
 
 func main() {
-	log.Printf("Beging terminating EC2 Instances")
-	terminateEc2Instances()
+	awsRegions := []string{"us-west-2","us-east-2"}
 
-	log.Printf("Begin destroy Load Balancer resources")
-	destroyLoadBalancerResource()
+	for _, region:= range awsRegions {
+		log.Printf("Beging terminating EC2 Instances %s", region)
+		terminateEc2Instances(region)
+
+		log.Printf("Begin destroy Load Balancer resources %s", region)
+		destroyLoadBalancerResource(region)
+
```

Comment:
nitpick: extra newline

Reply:
Delete extra newline. Fixed it with the newest commit.